docs/Contributing/getting-started/testing-and-local-development.md

```diff
@@ -1001,6 +1017,17 @@ Note that:
 3. To login into https://identity.apple.com/pushcert you can use your ABM account generated in the previous step.
 4. Save the token and certificate in a safe place.
 
+### Runing a local SCEP server
+
+To facilitate testing a variety of Fleet's features, it may be useful to set up a local SCEP certificate authority. To do so using a [MicroMDM SCEP server](https://github.com/micromdm/scep) on an arm64 macOS workstation:
```

Comment:
Do we need this if we already have this:
https://github.com/fleetdm/fleet/blob/main/docs/Contributing/product-groups/mdm/custom-scep-integration.md